code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -371,8 +371,8 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 			var/heat = conduction_coefficient*temperature_delta* \
 				(self_heat_capacity*sharer_heat_capacity/(self_heat_capacity+sharer_heat_capacity))
 
-			temperature = max(temperature - heat/self_heat_capacity, TCMB)
-			sharer_temperature = max(sharer_temperature + heat/sharer_heat_capacity, TCMB)
+			temperature -= heat/self_heat_capacity
+			sharer_temperature += heat/sharer_heat_capacity

```

Comment:
the checks removed here *should* be redundant and if removing them causes any problems it's a symptom of a greater underlying bug which should be fixed properly regardless